std/testing/bench.ts

```diff
@@ -18,10 +18,12 @@ export interface BenchmarkTimer {
 export interface BenchmarkFunction {
   (b: BenchmarkTimer): void | Promise<void>;
   name: string;
+  only?: boolean;
```

Comment:
Since `runBenchmarks` has an `only: Regexp` option too, I think it would be good to add JSDocs to these new `only: boolean` properties, so its clear what their function is.